.changeset/moody-rats-cover.md

```diff
@@ -0,0 +1,16 @@
+---
+"@fluid-experimental/presence": major
+---
+---
+"section": feature
+---
+
+Experimental Presence package added
+
+**[@fluid-experimental/presence](https://github.com/microsoft/FluidFramework/tree/main/packages/framework/presence#readme)** is now available for investigation. The new package is meant to support presence of collaborators connected to the same container. Use this library to quickly share simple, non-persisted data among all clients or send/receive fire and forget notifications.
```

Comment:
I'm not up to date with the exact capabilities of experimental presence, so I read the README, and wanted to confirm if the "_quickly share simple, non-persisted data among all clients_" has guaranteed delivery so all clients (eventually) agree on that state? Based on my little knowledge about experimental presence, I had it very tied to signals and non-guaranteed delivery. If that were the case, I'd suggest making it more explicit in the changeset, but I suspect that's just my outdated understanding.

Reply:
Good note to have. I'm adding reliability notes to readme. I think we are not quite as bad as fully unreliable as we think we isolated the most common drop situation and have done some accounting.